File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.py
class Solution:
    def removeStars(self, s: str) -> str:
        st=[]
        for i in s:
            if i=='*' and len(st)>0:
                st.pop(-1)
            else:
                st.append(i)
        h=''.join(st)
        return h
                
        

File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.cpp
class Solution {
public:
    string removeStars(string s) {
        stack<char>st;
        for(auto i: s){
            if(i=='*' && st.size()>0)st.pop();
            else st.push(i);
        }
        string res="";
        while(!st.empty()){
            res+=st.top();
            st.pop();
            
        }
        reverse(res.begin(),res.end());
    return res;
    }
};